### ISSUE TITLE
Authorizations are applied to search queries when using OIDC

### ISSUE THREAD

**Issue by Ben-Sheppard** [camunda/camunda#25992]

**Comment by tmetzke** [comment]
Leaving this to the identity stream and unassigning @nathansandi here.
@romansmirnov or @megglos will take care of assigning this accordingly.

**Comment by npepinpe** [comment]
This is done right?

**Comment by ThorbenLindhauer** [comment]
Yes, this topic is implemented (and the implementation specifics are different from the description here). General direction to look at for the tickets are https://github.com/camunda/camunda/issues/26799 and https://github.com/camunda/camunda/issues/26785.
